Minio.Tests/TestHelper.cs

```diff
@@ -23,6 +23,18 @@ class TestHelper
     {
         private static Random rnd = new Random();
 
+        public static string GetRandomObjectName(int length = 5)
```

Comment:
this function should have been added in  `Minio.Functional.Tests/FunctionalTest.cs` - functional tests and unit tests are in different namespaces here.